src/programs/project3d/project3d.cpp

```diff
@@ -225,13 +229,22 @@ bool Project3DApp::DoCalculation( ) {
     if ( project_based_on_star == true )
         number_of_projections_to_calculate = lines_to_process.GetCount( );
     else
+    {
         number_of_projections_to_calculate = global_euler_search.number_of_search_positions;
 
+        // setup paramters
+
+        output_params.parameters_to_write.SetActiveParameters(POSITION_IN_STACK | IMAGE_IS_ACTIVE | PSI | THETA | PHI | X_SHIFT | Y_SHIFT | DEFOCUS_1 | DEFOCUS_2 | DEFOCUS_ANGLE | PHASE_SHIFT | OCCUPANCY | LOGP | SIGMA | SCORE | PIXEL_SIZE | MICROSCOPE_VOLTAGE | MICROSCOPE_CS | AMPLITUDE_CONTRAST | BEAM_TILT_X | BEAM_TILT_Y | IMAGE_SHIFT_X | IMAGE_SHIFT_Y | ASSIGNED_SUBSET);
```

Comment:
Long logicals like this should be split one condition to a line, ending in the logical operator.

It is **very** easy to have a silent error if you forget one of the logical operators in certain circumstances.